plugins/gatsby-theme-iterative-docs/src/components/Documentation/Markdown/index.tsx

```diff
@@ -50,11 +50,21 @@ const Details: React.FC<{ slugger: GithubSlugger }> = ({
     firstChild.props.children.length - 1
   ) as ReactNode[]
 
-  let slug = slugger.slug(triggerChildren.toString())
-  if (slug[0] === '️') {
-    slug = slug.slice(1)
+  const title = (triggerChildren as any[]).reduce((acc, cur) => {
+    return (acc +=
+      typeof cur === 'string'
+        ? cur
+        : typeof cur === 'object'
+        ? cur?.props?.children?.toString()
+        : '')
+  }, '')
+
+  let slug = slugger.slug(title)
+  if (slug.includes('️')) {
+    slug = slug.replaceAll('️', '')
   }
```

Comment:
Does this do anything? As far as I understand from memory and a few quick tests, `string.includes('')` is always true, and `string.replaceAll('','')` always returns the string unchanged.

Reply:
There was an issue that made me scratch my head. It's with the `github-slugger` again. 
If you look into this you can see highlighting on my vscode snippet. It's not empty string but a character. 
<img width="669" alt="Screen Shot 2022-03-29 at 11 33 18" src="https://user-images.githubusercontent.com/20840228/160542094-0445c0d9-9126-4242-8644-7fcb059802bb.png">

> Not sure why, but links with ⚙️ are now not working correctly:
> 
> https://dvc-org-fix-details-obj-yq7a0c.herokuapp.com/doc/start/experiments#%EF%B8%8F-run-multiple-experiments-in-parallel

This means `github-slugger` generates that string for emoji and is not visible to the eye. So, that was the fix needed.